resources/s3/bucket_with_encryiption.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: s3.services.k8s.aws/v1alpha1
```

Comment:
recommend naming the file `resources/s3/v1alpha1/bucket-with-encryption.yaml` (note: spelling error on "encryption" and use hyphens instead of underscores to standardize *variants* using the `-with-` moniker.